packages/editor/src/store/selectors.js

```diff
@@ -454,18 +454,18 @@ export function isEditedPostSaveable( state ) {
  * @return {boolean} Whether post has content.
  */
 export function isEditedPostEmpty( state ) {
-	const blocks = getBlocksForSerialization( state );
+	const rootBlocks = getBlockOrder( state );
```

Comment:
It'd be more accurate to describe this as a set of client IDs, not blocks, i.e. `rootBlockClientIds`.